### DISCUSSION TITLE
Should strict=False be the default?

### DISCUSSION THREAD

**Discussion by MartinThoma** [py-pdf/pypdf#783] [General]
I've seen a lot of questions where the issue was essentially a broken PDF. The most recent case: https://github.com/py-pdf/PyPDF2/issues/672

It seems as if people don't know about the `strict` parameter.

I would like to hear your opinion on two ideas:

1. Make `strict=False` the default (in PyPDF2 2.0)
2. Change all documentation to use `strict=False` explicitly

**Comment by sekrause** [comment]
Yes `strict=False` would be a much better default, in almost all cases users will just want to have best-effort PDF processing without any exceptions.
